app/metrics.yaml

```diff
@@ -6229,7 +6229,8 @@ browser.search:
     type: labeled_counter
     description: |
       Records counts of SERP pages with adverts displayed.
-      The key format is `<provider-name>`.
+      The key format is 
+      `<provider-name>.in-content.[sap|sap-follow-on|organic].[code|none](.[channel])?`.
```

Comment:
Would you kindly expand a bit on the docs here? For example, please kindly document under which circumstances we can hit `none` (as flagged in [this comment](https://github.com/mozilla-mobile/firefox-android/pull/229#discussion_r1034534519)).

Would be relevant to also flag what `channel` is and under which circumstances it is present or absent.